src/main/scala-2.11/markets/participants/strategies/MarketOrderTradingStrategy.scala

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2015 David R. Pugh, J. Doyne Farmer, and Dan F. Tang
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package markets.participants.strategies
+
+import akka.agent.Agent
+
+import markets.tickers.Tick
+import markets.tradables.Tradable
+
+import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1704920069/issues/source?bid=2921361&fileBranchId=3046267#l23)